src/PotentialEnergyEquationTerms.jl

```diff
@@ -169,9 +143,9 @@ end
 
     grid = model.grid
     C = model.tracers
-    b = buoyancy_model.model
+    b_model = buoyancy_model.model
```

Comment:
This line doesn't read well. But I also don't have immediate suggestions on how to improve it... irrc the whole nomenclature of `BuoyancyModel` is kinda confusing and we discussed changing it over at Oceananigans a few times.

Maybe changing from `buoyancy_model` to `buoyancy_treatment` might help...?